index.html

```diff
@@ -1,8 +1,225 @@
+            <p>Alica Holla</p>
+          </div>
+        </div>
+        <div class="one-contributor">
+          <img src="pictures/contributors/Vanessa.png" alt="" class="contributor-picture">
+          <div class="name">
+            <a href="https://www.linkedin.com/in/vanessa-k%C3%B6ck-42a265244/" target="_blank" class="linkedin-button"><img src="pictures/LinkedinLogo.png" alt="linkedin link to the profile of Vanesa Köck" ></a>
+            <p>Vanessa Köck</p>
+          </div>
+        </div>
+        <div class="one-contributor">
+          <img src="pictures/contributors/Juju.png" alt="" class="contributor-picture">
+          <div class="name">
+            <a href="https://www.linkedin.com/in/juliana-matsumura/" target="_blank" class="linkedin-button"><img src="pictures/LinkedinLogo.png" alt="linkedin link to the profile of Juliana Matsumura" ></a>
+            <p>Juliana Matsumura</p>
+          </div>
+        </div>
+        <div class="one-contributor">
+          <img src="pictures/contributors/Antoine.png" alt="" class="contributor-picture">
+          <div class="name">
+            <a href="https://www.linkedin.com/in/antoine-muneret-5b81b9123/" target="_blank" class="linkedin-button"><img src="pictures/LinkedinLogo.png" alt="linkedin link to the profile of Antoine Muneret" ></a>
+            <p>Antoine Muneret</p>
+          </div>
+        </div>
+        <div class="one-contributor">
+          <img src="pictures/contributors/Hannah.png" alt="" class="contributor-picture">
+          <div class="name">
+            <a href="https://www.linkedin.com/in/hannah-wagner-4b9328268/" target="_blank" class="linkedin-button"><img src="pictures/LinkedinLogo.png" alt="linkedin link to the profile of Hannah Wagner" ></a>
+            <p>Hannah Wagner</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="github-container">
+      <img src="pictures/github.png" alt="github logo">
+      <p>Comixcraft is an open-source project and the repository is accessible on our <a href="https://github.com/comixcraft/Editor" target="_blank" rel="noopener noreferrer"> Github account</a>.</p>
+    </div>
+    <footer class="footer mobile-container">
+      <div class="top_row mobile-container">
+          <button class="logo-button" onclick="window.open('https://comixcraft.com/', '_blank')">
+            <img class="logo" src="/pictures/comixcraftLogo.svg" />
+          </button>
+              
+          <div class="link_container">
+              <p class="link_text" onclick="window.open('https://comixcraft.com/impresum', '_blank')" class="link_text">Impressum</p>
+              <p class="link_text" onclick="window.open('https://comixcraft.com/', '_blank')" class="link_text">comixcraft</p>
+              <p class="link_text" onclick="window.open('https://fhstp.github.io/comixplain/', '_blank')" >About Comixplain</p>
```

Comment:
These also should be a-tags. 
Also the link to the impressum is wrong -> you are missing a second s